docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,528 @@
+noti_ex_item_h text_handle = NULL;
+noti_ex_item_h image_handle = NULL;
+noti_ex_item_h time_handle = NULL;
+time_t current_time;
+
+ret = noti_ex_item_group_create(&group_item, "group_id");
+
+noti_ex_item_text_create(&name_handle, "name_id", "name", NULL);
+noti_ex_item_text_create(&text_handle, "text_id", "text", NULL);
+noti_ex_item_image_create(&image_handle, "image_id", APP_IMAGE_FULL_PATH);
+
+time(&current_time);
+noti_ex_item_time_create(&time_handle, "time_id", current_time);
+
+ret = noti_ex_item_chat_message_create(&chatmessage_item, "message_id", name_handle,
+	text_handle, image_handle, time_handle, NOTI_EX_ITEM_CHAT_MESSAGE_TYPE_USER);
+if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+
+ret = noti_ex_item_group_add_child(group_item, chatmessage_item);
+if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+```
+
+<a name="reporter"></a>
+## Make a Notification-ex Reporter Handle
+
+To send a notification, you need to create a handle for the reporter. When creating a handle, you can register an error callback and an event callback.
+```
+noti_ex_reporter_h handle;
+noti_ex_reporter_events_s ev;
+
+ev.event = _reporter_events_event_cb;
+ev.error = _reporter_events_error_cb;
+
+ret = noti_ex_reporter_create(&handle, ev, NULL);
+if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+```
+
+<a name="post"></a>
+## Posting a Notification
+
+You can post a notification and also post it as a list.
+
+To post a notification, use the `noti_ex_reporter_post()` function:
+```
+ret = noti_ex_reporter_post(handle, group_item, &request_id);
+if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+```
+
+To post the notification list, use the `noti_ex_reporter_post_all()` function:
+```
+ret = noti_ex_reporter_post_list(handle, noti_list, 2, &request_id);
+if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+```
+<a name="update"></a>
+## Updating Notification
+
+To update the content of a notification, use the `noti_ex_reporter_update()` function.
+```
+ret = noti_ex_reporter_update(handle, group_item, &request_id);
+if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+```
+
+<a name="delete"></a>
+## Deleting a Notification
+
+To delete a notification:
+
+1. To delete a notification from the database, use the `noti_ex_reporter_delete()` function:
+  ```
+  ret = noti_ex_reporter_delete(handle, group_item, &request_id);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+2. After deleting the notification, free the internal structure data of the notification handle by calling the `noti_ex_item_destroy()` function:
+  ```
+  ret = noti_ex_item_destroy(group_item);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+To delete all notifications:
+
+1. To delete all notifications from the database, use the `noti_ex_reporter_delete_all()` function:
+  ```
+  ret = noti_ex_reporter_delete_all(handle, &request_id);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
```

Comment:
Would you please add the supported versions here?
For example 

\## Related Information
\- Dependencies
 &nbsp; &nbsp; \- Tizen 5.5 and Higher for Mobile
  &nbsp; &nbsp;  \- Tizen 5.5 and Higher for Wearable